os_migrate/playbooks/export_networks.yml

```diff
@@ -1,3 +1,6 @@
 - hosts: migrator
+  environment:
+    OS_CLIENT_CONFIG_FILE: "{{ os_migrate_data_dir }}/clouds.yaml"
```

Comment:
This should work, but as we talked before about configurability of this, we may consider adding a `os_migrate_clouds_path` variable and using the above as default. So it would become:

```
OS_CLIENT_CONFIG_FILE: "{{ os_migrate_clouds_path|default(os_migrate_data_dir ~ '/clouds.yaml') }}"
```